client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -9,17 +9,63 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
+import Card from 'components/card';
 import FormattedHeader from 'components/formatted-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextarea from 'components/forms/form-textarea';
+import FormTextInput from 'components/forms/form-text-input';
+import Main from 'components/main';
 
 class JetpackOnboardingSiteTitleStep extends React.PureComponent {
+	state = {
+		description: '',
+		title: '',
+	};
+
+	setDescription = event => {
+		this.setState( { description: event.target.value } );
+	};
+
+	setTitle = event => {
+		this.setState( { title: event.target.value } );
+	};
+
 	render() {
 		const { translate } = this.props;
 		const headerText = translate( "Let's get started." );
 		const subHeaderText = translate(
 			'First up, what would you like to name your site and have as its public description?'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		return (
+			<Main className="jetpack-onboarding-steps__site-title">
```

Comment:
Something else I think we missed (it's missing in the prototype too) is a `<DocumentHead />` - we should be setting a document title, perhaps **Jetpack Onboarding -- WordPress.com** or **Site Title < Jetpack Onboarding -- WordPress.com**.